### PR TITLE
Rebuild schema to remove the cached removed field

### PR DESCRIPTION
**Pull request by evgenykuzyakov**

### CODE REVIEW

**Comment by evgenykuzyakov** [comment]
Looks like there are name conflicts for superclass parsing. How do I solve this?

**Comment by vgrichina** [comment]
Why everything is getting re-ordered?

**Comment by evgenykuzyakov** [comment]
I've rebuild the schema from scratch. Have to nuke `VMError` from errors

File: runtime/near-vm-errors/src/lib.rs [hunk 1]
```diff
@@ -3,9 +3,7 @@ use near_rpc_error_macro::RpcError;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
-#[derive(
-    Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Deserialize, Serialize, RpcError,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Deserialize, Serialize)]
```
Comment (nearmax) [inline runtime/near-vm-errors/src/lib.rs]: Why are we removing `RpcError`? Don't we want to generate error schema for it?

Comment (evgenykuzyakov) [reply runtime/near-vm-errors/src/lib.rs]: Yes. It's not exposed outside of the Runtime. Inner fields are converted to either `StorageError` or to `FunctionCallError`

**Comment by evgenykuzyakov** [comment]
I'm increasing version of `staging` to `0.4.13` to avoid having API conflicts with `nearlib` comparing to nearcore `master` branch

**Pull request merged**